source/client/renderer/GameRenderer.cpp

```diff
@@ -329,7 +329,7 @@ void GameRenderer::setupFog(int i)
 	fog_color[3] = 1.0f;
 
 	glFogfv(GL_FOG_COLOR, fog_color);
-	glNormal3f(0.0f, -1.0f, 0.0f);
+	//glNormal3f(0.0f, -1.0f, 0.0f);
```

Comment:
Why?

Reply:
This doesn't do anything without `GL_LIGHTING`, and that is never enabled.